.github/workflows/trusted-release-workflow.yml

```
@@ -104,18 +105,79 @@ jobs:
+          subject-path: ./dist/${{ steps.checksumtxt.outputs.checksum_file }}
 
-  release:
+  binstaller:
     needs: [version, goreleaser]
     if: needs.version.outputs.tag_name != ''
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write # needed for keyless signing
+      attestations: write # needed for provenance
+    outputs:
+      artifacts: ${{ steps.goreleaser.outputs.artifacts }}
```

actionlint (1.7.7)

property "goreleaser" is not defined in object type {check-config: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)